### PR TITLE
["dash"] Improve Dash styling

### PR DESCRIPTION
**Pull request by ryon**
## Fixes issues

- #315 
- #313 
- #310 
- #306 

## Other improvements

- Switch to Open Sans per Penn branding guidelines
- Fix width of sidebar
- Use `.container`s to limit width of body text in main content area while leaving charts full width
- Wrangled date picker component to fit in with Bootstrap inputs

## Things I'm not sure about ⚠️ 

I made changes in https://github.com/CodeForPhilly/chime/commit/17964dc66efec49b035356f4653d72e8718733ac to enable live reloading, and because the files in `assets/` weren't loading. But I'm not sure if those changes might break something else.

### Before

dash-before

### After

dash-after

### CODE REVIEW

**Review by ecsmyth — CHANGES_REQUESTED** [review]
Hey @ryon, changes look good. I pulled your changes and fixed the conflicts with the PR of mine that landed earlier today. I pushed them to the `JarvusInnovations/dash` branch.

For future reference, when using the style property on the Dash components, the keys must be React format (i.e. camelCase "marginLeft") vs using the css property (i.e. "margin-left").

**Comment by jlubken** [comment]
Can I get a review on the changes to the `dash_app.py` file? I suspect that debug should not be set to True. @quinn-dougherty @ckoerber

**Comment by quinn-dougherty** [comment]
Brian pushed this branch to a deploy last night and it's messed up for us https://btr-test.herokuapp.com/ . the header text is the same color as the header background, and (pictured) the date input widget is off
Screenshot from 2020-03-31 21-40-31

i'm using Firefox

However, running locally it looks good, so we'll need to do more wrestling with heroku. its possible heroku just isn't dealing with the static files dir?

File: src/chime_dash/app/run.py [hunk 1]
```diff
@@ -13,7 +13,6 @@
 SCRIPTS     = Body(LANGUAGE, DEFAULTS).external_scripts
 
 DASH = Dash(
-    __name__,
     external_stylesheets=STYLESHEETS,
```
Comment (quinn-dougherty) [inline src/chime_dash/app/run.py]: could you please replace __name__ here? i'm not 100% clear why it appears to work just as well either way, but as we start to debug the heroku deploy we'll want it to be in there

File: src/dash_app.py [hunk 2]
```diff
@@ -41,4 +41,4 @@ def callback(*args):  # pylint: disable=W0612
 
 if __name__ == "__main__":
     #    main()
-    DASH.run_server(host="0.0.0.0")
+    DASH.run_server(host="0.0.0.0", debug=True)
```
Comment (quinn-dougherty) [inline src/dash_app.py]: per Jason's comment, please remove `debug=True` from here

**Review by BrianThomasRoss — COMMENTED** [review]
When making changes to styling please refrain from changes to code outside that scope

**Comment by quinn-dougherty** [comment]
static files are fine in our test deploy. disregard my previous comment.

**Review by BrianThomasRoss — COMMENTED** [review]
Seems to be working now merged into testing branch, on successful deploy there will go ahead an merge into stable

**Review by BrianThomasRoss — APPROVED** [review]

**Comment by BrianThomasRoss** [comment]
> Can I get a review on the changes to the `dash_app.py` file? I suspect that debug should not be set to True. @jlubken 

debug=True only in development or testing

**Review by ecsmyth — APPROVED** [review]

**Comment by ecsmyth** [comment]
> 
> 
> > Can I get a review on the changes to the `dash_app.py` file? I suspect that debug should not be set to True. @jlubken
> 
> debug=True only in development or testing

No longer an issue. When I fixed the conflicts with my callback refactor, I fixed the `asset` path and removed `debug=True`. The comment should be removed, but probably not worth blocking the PR over.

**Review by BrianThomasRoss — APPROVED** [review]
Thanks again for the contribution, really gives it a nice clean look. In the future please submit PRs from a feature branch (e.g. dash-improved-stylings)

**Pull request merged**